recipe/meta.yaml

```
@@ -68,36 +74,37 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
-    # Dec 2020: it seems that git is broken on windows, so we use m2-git
     - m2-patch  # [win]
-    - m2-git    # [win]
+    - m2-sed    # [win]
     - patch     # [not win]
     - git       # [not win]
     - libgomp        # [linux]
     - llvm-openmp    # [osx]
```

Do we want and/or need an openmp-implementation here? Fine if this is deferred, but just thought I'd point it out.
```suggestion
    - libgomp        # [linux]
    - llvm-openmp    # [osx or win]
```

---

See https://github.com/baszalmstra/pytorch-cpu-feedstock/blob/8fe95717d923bb89483c967b72a2ace8cceaa82e/recipe/bld.bat#L113C1-L114C19 - I'd prefer to defer this for later.

---

Related: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/302#issuecomment-2544162590